tests/test_inbox.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+from flask import url_for
+from flask.testing import FlaskClient
+
+from hushline.db import db
+from hushline.model import Message, User, Username
+
+
+@pytest.mark.usefixtures("_authenticated_user")
+def test_delete_message(client: FlaskClient, user: User) -> None:
+    # Setup: create primary and alias usernames, and a message for the user
+    primary_username = Username(
```

Comment:
Usernames and messages should be created via fixture since there's not any special logic here.

Reply:
done